packages/react/src/tests/MessageRepository.test.ts

```
@@ -0,0 +1,690 @@
+        {
+          role: "assistant" as const,
+          content: [
+            { type: "text" as const, text: "Hi there" },
+          ] as TextContentPart[],
+        },
+      ];
+
+      const result = ExportedMessageRepository.fromArray(messages);
+
+      expect(result.messages).toHaveLength(2);
+      expect(result.messages[0]!.parentId).toBeNull();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)